### DISCUSSION TITLE
regarding #6119

### DISCUSSION THREAD

**Discussion by TakayamaYoshimichi** [vercel/community#6122] [Feature Question] [answered]
### Question

regarding #6119 
I have one more question.
How much does it cost to have a consultation service?
Basically, what we want to do can be achieved with a PRO plan.
We are aware that we almost never need to consult with a consultation service, only that it is necessary for company procedures.

**Comment by TakayamaYoshimichi** [comment]
@swarnava 
Could you answer above question?

**Comment by amyegan** [comment, marked as answer]
Hi @TakayamaYoshimichi. Please [contact our sales team](https://vercel.com/contact/sales) to determine the best options for your team. They know all of the service options and pricing details, so they will be best able to answer your questions.
